docs/pages/guides/using-sentry.mdx

```diff
@@ -52,146 +63,22 @@ Once you have each of these: **organization slug**, **project name**, **DSN**, a
-
-```tsx app/_layout.tsx
-import { Slot, useNavigationContainerRef } from 'expo-router';
-import { useEffect } from 'react';
-import * as Sentry from '@sentry/react-native';
-import { isRunningInExpoGo } from 'expo';
-
-// Construct a new integration instance. This is needed to communicate between the integration and React
-const navigationIntegration = Sentry.reactNavigationIntegration({
-  enableTimeToInitialDisplay: !isRunningInExpoGo(),
-});
-
-Sentry.init({
-  dsn: 'YOUR DSN HERE',
-  debug: true, // If `true`, Sentry will try to print out useful debugging information if something goes wrong with sending the event. Set it to `false` in production
-  tracesSampleRate: 1.0, // Set tracesSampleRate to 1.0 to capture 100% of transactions for tracing. Adjusting this value in production.
-  integrations: [
-    // Pass integration
-    navigationIntegration,
-  ],
-  enableNativeFramesTracking: !isRunningInExpoGo(), // Tracks slow and frozen frames in the application
-});
-
-function RootLayout() {
-  // Capture the NavigationContainer ref and register it with the integration.
-  const ref = useNavigationContainerRef();
-
-  useEffect(() => {
-    if (ref?.current) {
-      navigationIntegration.registerNavigationContainer(ref);
-    }
-  }, [ref]);
-
-  return <Slot />;
-}
-
-// Wrap the Root Layout route component with `Sentry.wrap` to capture gesture info and profiling data.
-export default Sentry.wrap(RootLayout);
-```
+### Use the Sentry wizard to set up your project
 
-</Tab>
+The easiest way to set up Sentry in your Expo project is to use the Sentry wizard. This tool will automatically configure your project with the right settings.
 
-<Tab label="Not using Expo Router">
+Run the following command in your project directory:
 
-Add the following to your app's main file such as **App.js**:
+<Terminal cmd={['$ npx @sentry/wizard@latest -i reactNative']} />
 
-```js
-import * as Sentry from '@sentry/react-native';
+The wizard will:
 
-Sentry.init({
-  dsn: 'YOUR DSN HERE',
-  debug: true, // If `true`, Sentry will try to print out useful debugging information if something goes wrong with sending the event. Set it to `false` in production
-});
-```
-
-Now wrap the root component of your app with Sentry. This may be **App.js** or **index.js** depending on your project.
+- Install the required dependencies
+- Configure your project to use Sentry
+- Set up the Metro configuration automatically
+- Add the necessary initialization code to your app
 
-```js
-import * as Sentry from '@sentry/react-native';
-
-// Your App component here
-
-export default Sentry.wrap(App);
-```
-
-</Tab>
-
-</Tabs>
+Follow the prompts in the wizard to complete the setup process. The wizard will ask for your **organization slug**, **project name**, and **auth token** that you noted earlier.
```

Comment:
```suggestion
Follow the prompts in the wizard to complete the setup process. The wizard will guide you to log in to your Sentry account and fetch all the correct information regarding your project.
```

Reply:
@amandeepmittal we can write it like this instead.

Reply:
Yes, that sounds perfect.